### ISSUE TITLE
Travis fails on Electron builds using Node 12.x

### ISSUE THREAD

**Issue by pdesantis** [mapbox/node-sqlite3#1197]
The error is:
```
Error: No test files found
The command "if [[ "${NODE_VERSION}" ]]; then if [[ "${ELECTRON_VERSION}" ]]; then ./scripts/build_against_electron.sh; else ./scripts/build_against_node.sh; fi; fi" exited with 1.
```

This seems to affect all new PRs, such as #1192 & #1195

Here is a failing Travis job: https://travis-ci.org/mapbox/node-sqlite3/jobs/565236851

**Comment by kewde** [comment]
Thank you for reporting the issue, I am resolving this.

**Comment by kewde** [comment]
c61ab026f5a8084683606c77eb37bf8a83717b00

**Comment by kewde** [comment]
Solved in #1195
